specification/web/resource-manager/Microsoft.Web/stable/2021-03-01/WebApps.json

```diff
@@ -3523,6 +3523,88 @@
           }
         }
       }
+    },
+	  "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/extensions/onedeploy": {
+      "get": {
+        "tags": [
+          "WebApps"
+        ],
+        "summary": "Invoke onedeploy status API /api/deployments and gets the deployment status for the site",
+        "description": "Description for Invoke onedeploy status API /api/deployments and gets the deployment status for the site",
+        "operationId": "WebApps_GetOneDeployStatus",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of web app.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "object"
+            }
```

Comment:
There is no properties in this schema?

Reply:
currently, the Get call returns the json object type, is it required to have the typed schema return type?

Reply:
Almost every API in ARM returns a json object.

If your json object does have a fixed format, yes, it is required.

Reply:
I see, let me work on to add it.